### DISCUSSION TITLE
怎样设置VcDrawings绘制billboardOpts动态图片

### DISCUSSION THREAD

**Discussion by OoOoYY** [zouyaoji/vue-cesium#295] [Q&A] [answered]
求助大佬，怎样设置VcDrawings绘制billboardOpts动态图片

**Comment by zouyaoji** [comment, marked as answer]
先看看这个
https://github.com/zouyaoji/vue-cesium/issues/167

**Reply by OoOoYY** [reply]
需求是实现绘制gif动图（或者理解为地图上加一些动效），我用libgif.js+Cesium.CallbackProperty试了下没成功

**Reply by zouyaoji** [reply]
自己解析gif，更改材质的图片应该是可以的。
